contracts/Organization.sol

```diff
@@ -0,0 +1,272 @@
+     *  Expiration height 0 is a valid height.
+     */
+    mapping(address => uint256 /* expiration height */) public workers;
+
+
+    /* Modifiers */
+
+    modifier onlyOwner()
+    {
+        require(
+            msg.sender == owner,
+            "Only owner is allowed to call."
+        );
+        _;
+    }
+
+    modifier onlyOrganization()
+    {
+        require(
+            (msg.sender == owner) || (msg.sender == admin),
+            "Only owner/admin is allowed to call."
+        );
+        _;
+    }
+
+
+    /* Special Functions */
+
+    constructor() public
+    {
+        owner = msg.sender;
+    }
+
+
+    /* External Functions */
+
+    /**
+     * @notice Initiates ownership transfer to proposed owner.
+     *
+     * @dev Requires:
+     *          - msg.sender should be owner.
+     *      Allows resetting of owner to 0x address.
+     *
+     * @param _proposedOwner worker address to be added.
+     *
+     * @return true on successful execution.
+     */
+    function initiateOwnershipTransfer(address _proposedOwner)
+        external
+        onlyOwner
+        returns (bool)
+    {
+        require(
+            _proposedOwner != owner,
+            "ProposedOwner address can't be current owner address."
+        );
+
+        proposedOwner = _proposedOwner;
+
+        emit OwnershipTransferInitiated(_proposedOwner);
+
+        return true;
+    }
+
+    /**
+     * @notice Complete ownership transfer to proposed owner.
+     *
+     * @dev Requires:
+     *          - msg.sender should be proposed owner.
+     *
+     * @return true on successful execution.
+     */
+    function completeOwnershipTransfer() external returns (bool)
+    {
+        require(
+            msg.sender == proposedOwner,
+            "Caller is not proposed owner address."
+        );
+
+        address oldOwner = owner;
+        owner = proposedOwner;
+        proposedOwner = address(0);
+
+        emit OwnershipTransferCompleted(oldOwner, owner);
+
+        return true;
+    }
+
+    /**
+     * @notice Sets admin address.
+     *
+     * @dev Requires:
+     *          - msg.sender should be owner or admin.
+     *          - admin should not be same as owner.
+     *      Allows resetting of admin address to 0x.
+     *
+     * @param _admin admin address to be added.
+     *
+     * @return true on successful execution.
+     */
+    function setAdmin(address _admin) external onlyOrganization returns (bool)
+    {
+        require(
+            _admin != owner,
+            "Admin address can't be owner address."
+        );
+
+        admin = _admin;
+
+        emit AdminAddressChanged(admin);
+
+        return true;
+    }
+
+    /**
+     * @notice Sets worker and its expiration height.
+     *
+     * @dev Requires:
+     *          - Caller should be owner or admin.
+     *          - worker address can't be null.
+     *          - expiration height should not be expired.
+     *          - 0 expiration height is not allowed.
```

Comment:
It's helpful for non developers for sure.
Also I think it's a nice way to represent validations like other contracts:
https://github.com/OpenSTFoundation/openst-contracts/blob/develop/contracts/TokenRules.sol#L150

Reply:
To clarify, what I am doubting here is the value of stating that 0 is not an acceptable expiration height.

In the example you point to, each one of those documented requirements is also an explicitly-coded requirement. In this case, however, there is no explicitly coded requirement against 0; it is implicitly the case that 0 is not allowed because the min. possible block number is 0 and the expiration height must be greater than the current block number.

My point is that I'm not sure there is value in explicitly pointing out this implicit requirement given what you point out in the other example (that we only name the explicit requirements) and that the practical reality is that this will rarely present an issue in the case of proper interaction with this function.